Header header
uint8[] cell_num
uint8[] rec_level
uint8[] rec_quality
